### PR TITLE
Fix URL to Ansible Galaxy User guide

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)

### CODE REVIEW

File: guides/common/modules/proc_synchronizing-ansible-collections.adoc [hunk 1]
```diff
@@ -39,7 +39,7 @@ collections:
   version: 1.2.3
 ----
 +
-For more information, see https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#install-multiple-collections-with-a-requirements-file[Install multiple collections with a requirements file] in _Galaxy User Guide_.
+For more information, see https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-roles-and-collections-from-the-same-requirements-yml-file[Installing roles and collections from the same requirements.yml file] in _Galaxy User Guide_.
```
Comment (Lennonka) [inline guides/common/modules/proc_synchronizing-ansible-collections.adoc:42]:
```suggestion
For more information, see https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-roles-and-collections-from-the-same-requirements-yml-file[Installing roles and collections from the same requirements.yml file] in the _Galaxy User Guide_.
```

**Review by Lennonka — APPROVED** [review]
Would you mind updating it in all maintained versions?

File: guides/common/modules/proc_synchronizing-ansible-collections.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_synchronizing-ansible-collections.adoc:42]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Rebased to "master" and applied feedback by Lena. Will merge with green GHA.

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   076a35c6f4..dba651a07b  3.10 -> 3.10
   90269accec..9815a9899b  3.9 -> 3.9
   06f9b0dca8..7f7700088f  3.8 -> 3.8
   992384fa99..15548f77a4  3.7 -> 3.7
   c88c0188ff..8ed86813e1  3.6 -> 3.6
   c4ec7084d4..124ec1dc35  3.5 -> 3.5
   64e1f6c207..4cffa1f72f  3.4 -> 3.4
   7f81ac67f7..c997336f92  3.3 -> 3.3

**Pull request merged**